src/assets/scripts/client/aircraft/FlightManagementSystem/RouteModel.js

```diff
@@ -772,26 +772,17 @@ export default class RouteModel extends BaseModel {
      * @return {boolean} whether operation was successful
      */
     replaceDepartureProcedure(routeString) {
-        let sidLegModel;
+        let sidModel;
```

Comment:
It's not an instance of the SID definition, but rather an instance of a route which has only the SID leg. Thus, we should call this `sidRouteModel`. You can tell that will be a good name when we look down below and see we would then have `this.absorbRouteModel(sidRouteModel)`.

Reply:
done